core/src/main/scala/akka/kafka/internal/KafkaConsumerActor.scala

```diff
@@ -224,6 +224,23 @@ import scala.util.control.NonFatal
   private var commitsInProgress = 0
```

Comment:
this member should be volatile, because it will be update in commitAsync callback.

Reply:
Looks, like it, but that is not required as it will always be called via `poll()` which is the actor's thread:
```
/**
 * A callback interface that the user can implement to trigger custom actions when a commit request completes. The callback
 * may be executed in any thread calling {@link Consumer#poll(java.time.Duration) poll()}.
 */
public interface OffsetCommitCallback {
```